include/aos/common/tools/log.hpp

```diff
@@ -189,6 +316,7 @@ class Log : private NonCopyable {
     }
 
     StaticString<cMaxLineLen> mLogLine;
+    StaticString<cMaxLineLen> mFieldsLine;
```

Comment:
We can use just field counter and put values to `mLogLine` directly,

Reply:
In this case we could not use log like this 
```
LOG_DBG() << "Downloaded" << Log::Field("path", path) << " size" << Log::Field("size", fileSize);
```

output 

```
 "Downloaded size: path=/hello/world, size=20"
```

if we will be use only mLogLine in this case will be diffucult to put " size" after "Downloaded"